opentracing-api/src/main/java/io/opentracing/Tracer.java

```diff
@@ -18,7 +18,19 @@
 /**
  * Tracer is a simple, thin interface for Span creation and propagation across arbitrary transports.
  */
-public interface Tracer extends ActiveSpanSource {
+public interface Tracer {
+
+    /**
+     * @return the current {@link ScopeManager}, which may be a noop but may not be null.
+     */
+    ScopeManager scopeManager();
+
+    /**
+     * Set a new {@link ScopeManager}, presumably as part of initialization. The state of any already active
+     * {@link Scope} instance after this is only defined to do no harm to the process (i.e., not-crash). Again, this is
+     * only intended for use during initialization.
+     */
+    void setScopeManager(ScopeManager scopeManager);
```

Comment:
It would be nice not to require that in the API, since it can always be a part of implementation-specific initialization sequence. Do we know of a use case where this method is absolutely required in the Tracer API?